lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -56,6 +44,30 @@ extension String {
   }
 }
 
+extension SyntaxProtocol {
+  /// Obtains the bridged start location of the node in the source buffer provided by `astgen`.
```

Comment:
Could you please state that the location excludes leading trivia, as the old comment did?